aries-site/src/data/structures/components.js

```diff
@@ -765,6 +765,7 @@ export const components = [
     relatedContent: ['Layer', 'Lists', 'Tabs'],
     seoDescription:
       'The accordion is a list with expanded and collapsed states. When clicked opened, it reveals more detail. See different behaviors, states, and icons.',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
```suggestion
    accessibility: 'Passed WCAG 2.2 AAA',
```
It looks like no AAA rules are failing so I think we could update this to say that we are passing AAA